### ISSUE TITLE
Nothing happen on "live" images uploading on map marker

### ISSUE THREAD

**Issue by killianweid** [morehawes/Waymark#27]
When I try to upload an image taken at the moment into a marker popup nothing happens..

**Comment by morehawes** [comment]
Hi @killianweid,

Thanks for reaching out. Could you provide any extra information about your setup (like device, browser, example URL or screenshots etc) so that I can replicate on my end?

Perhaps you are trying to upload from a smartphone camera directly into the Waymark Editor?

Joe

**Comment by killianweid** [comment]
The problem is on every browsers on iPhone (I don’t try on Android because I don’t have).
And yes I try to upload from my iPhone camera into the Waymark Editor.

Best regards,
Killian Weidner

Le 14 nov. 2023 à 00:45, Joe Hawes ***@***.***> a écrit :

﻿

Hi @killianweid<https://github.com/killianweid>,

Thanks for reaching out. Could you provide any extra information about your setup (like device, browser, example URL or screenshots etc) so that I can replicate on my end?

Perhaps you are trying to upload from a smartphone camera directly into the Waymark Editor?

Joe

**Comment by killianweid** [comment]
Hello @morehawes, problem is still not resolved ?
Thanks

**Comment by morehawes** [comment]
Hi @killianweid,

Is this **only** happening on mobile when uploading a photo taken directly from the camera? That is, not uploading a photo already saved on the device?

Does it work correctly if you select a previously taken photo for upload?

Cheers,

Joe

**Comment by killianweid** [comment]
@morehawes Yes it's only happening when uploading a photo taken directly from the camera. No problem when I select a previously taken photo for upload.

Best regards,
Killian

**Comment by morehawes** [comment]
@killianweid 

OK understood, I can confirm I am seeing the same thing so this is indeed a bug. However I am having a hard time diagnosing the issue, as I am not getting any errors in the iPhone browser error console.

As there seems to be a simple work-around (i.e. saving a photo to the device and then selecting for upload) and you are the first to raise this, it might be a while until this gets to the top of my list.

Thanks again for raising this and I will be sure to update here as/when this changes.
